src/blacken_docs/__init__.py

```diff
@@ -293,10 +297,14 @@ def main(argv: Sequence[str] | None = None) -> int:
 
     retv = 0
     for filename in args.filenames:
-        retv |= format_file(
-            filename,
-            black_mode,
-            skip_errors=args.skip_errors,
-            rst_literal_blocks=args.rst_literal_blocks,
+        retv = max(
+            retv,
+            format_file(
```

Comment:
Hello, I opted for max was because this operator felt a little weird.

(Initially I even thought this was a "assign if not false" actually - I'd never heard of the bit-wise assignment so thanks 🙂)

Knowing that this uses bit operations has me more convinced that a `max` would be the simplest and clearest solution - if you're anything like me you learned about bit operations in school, promptly forgot about them, and now struggle to remember them once every two years when they pop up in the wild.

The current possible non-zero exit values for `blacken-docs` are 1, when some file needs to be changed and 2, when black itself has an error.

To me it made more sense to simply exit with the most severe error.

Consider:

```bash
$ blacken-docs file_that_needs_a_rewrire file_that_has_syntax_errors
```

With the `|=` bash would exit with 3; which doesn't have any meaning.

If we were to add more exit values (eg: 3,4,5) it would get even more complex.

```bash
$ blacken-docs file_that_exits_with_3 file_that_exits_with_5
# program exits with error 7
$ blacken-docs file_that_exits_with_2 file_that_exits_with_5
# also exits with 7
```

The docs only mention that we exit with a non-zero; so I might be thinking too much about something that doesn't matter much.

I'll certainly revert if you have strong feelings on the matter.

Reply:
> With the |= bash would exit with 3; which doesn't have any meaning.

It does have meaning, it means 1 and 2 at the same time, for different files.

I’m gonna revert back to `|=`